test/fs/test_nodefs_readdir.out

```diff
@@ -0,0 +1,13 @@
+listing contents of dir=/
+.
+..
+tmp
+home
+dev
+proc
+listing contents of dir=/working
+existing
```

Comment:
Strange that we have `.` and `..` at the root but not in the subdirectory.    I guess that just a bug.